typeql/language/insert.feature

```diff
@@ -2772,3 +2772,35 @@ Parker";
       $x isa bird;
       $x iid V123;
       """
+
+
+  Scenario: datetime object can be inserted and queried correctly (and time-zone-naively) regardless of OS timezone
```

Comment:
If this test is basically doing the same thing as the other test, why is it named so differently?

Also, this should go under the `Attribute Ownership` section right? 'Edge Cases' is something we should do our best to stay away from as a category otherwise it'll just get filled up very quickly when people get too lazy to properly categorise the tests

cc @krishnangovindraj